tools/docker/builder/Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN apt-get update && apt-get install -y \
     python-yaml \
     python3 \
     python3-yaml \
+    uuid-runtime \
```

Comment:
I assume this should not only be used by docker - so please add it to the main readme as well

Reply:
Since the script will tell you to install uuidgen if you don't have it, I don't think it's needed to add it to the main readme. Anyway, nobody outside of the maintainer team is going to use that script.